viewer/src/__tests__/views/threejs/cad/primitives.test.ts

```diff
@@ -4,123 +4,204 @@
 
 import { Sector } from '../../../../models/cad/types';
 import { createPrimitives } from '../../../../views/threejs/cad/primitives';
-import { PrimitiveAttributes } from '../../../../workers/types/parser.types';
 import { createEmptySector } from '../../../models/cad/emptySector';
-import { createMaterials } from '../../../../views/threejs/cad/materials';
+import { createMaterials, Materials } from '../../../../views/threejs/cad/materials';
+import { ParsePrimitiveAttribute } from '../../../../workers/types/parser.types';
+import { Mesh, InstancedBufferGeometry, LOD } from 'three';
 import { RenderMode } from '../../../../views/threejs/materials';
 
+function createMockAttributes(): Map<string, ParsePrimitiveAttribute>{
+ 
+  let map = new Map<string, ParsePrimitiveAttribute>();
+  
+  let mockAttribute_0 = {offset: 0, size: 4} as ParsePrimitiveAttribute;
+  let mockAttribute_1 = {offset: 4, size: 8} as ParsePrimitiveAttribute;
+  let mockAttribute_2 = {offset: 12, size: 12} as ParsePrimitiveAttribute;
```

Comment:
`let mockAttribute_0: ParsePrimitiveAttribute = {offset: 0, size: 4}` ensures you fulfill the type - `aas ParsePrimitiveAttribute` casts (i.e. is less safe)